doc/man/scons.xml

```diff
@@ -651,32 +652,25 @@ before searching for the
 or
 <emphasis>sconstruct.py</emphasis>
 file, or doing anything
-else.  Multiple
-<option>-C</option>
-options are interpreted
-relative to the previous one, and the right-most
+else.  When multiple
 <option>-C</option>
-option wins. (This option is nearly
-equivalent to
-<option>-f directory/SConstruct</option>,
-except that it will search for
-<emphasis>SConstruct</emphasis>,
-<emphasis>Sconstruct</emphasis>,
-<emphasis>sconstruct</emphasis>,
-<emphasis>SConstruct.py</emphasis>
-<emphasis>Sconstruct.py</emphasis>
-or
-<emphasis>sconstruct.py</emphasis>
-in the specified directory.)</para>
+options are given, each subsequent non-absolute
+<option>-C</option> <replaceable>directory</replaceable>
+is interpreted relative to the preceding one.
+This option is similar to
+<option>-f <replaceable>directory</replaceable>/SConstruct</option>,
+but <option>-f</option> does not search for any of the
+predefined <emphasis>SConstruct</emphasis> names
+in the specified directory.</para>
+  </listitem>
+  </varlistentry>
 
 <!--  .TP -->
 <!--  \-d -->
 <!--  Display dependencies while building target files.  Useful for -->
 <!--  figuring out why a specific file is being rebuilt, as well as -->
 <!--  general debugging of the build process. -->
 
-  </listitem>
```

Comment:
Where's the matching /listitem ?

Reply:
what happened here is the commented-out and unused -d option was moved to be in between list entries, rather than inside the -C item.  `</listitem>` was not removed, it's at line 665.

Reply:
Ahh see it now!